docs/user-guide/ecosystem.md

```diff
@@ -12,9 +12,11 @@ On this page you can find a non-exhaustive list of libraries and tools that supp
 - [Machine learning](#machine-learning)
 - [Other](#other)
 
-#### [Apache Arrow](https://arrow.apache.org/)
+---
 
-Apache Arrow enables zero-copy reads of data within the same process, meaning that data can be directly accessed in its in-memory format without the need for copying or serialisation. This enhances performance when integrating with different tools using Apache Arrow. Polars is compatible with a wide range of libraries that also make use of Apache Arrow, like Pandas and DuckDB.
+### Apache Arrow
+
+[Apache Arrow](https://arrow.apache.org/) enables zero-copy reads of data within the same process, meaning that data can be directly accessed in its in-memory format without the need for copying or serialisation. This enhances performance when integrating with different tools using Apache Arrow. Polars is compatible with a wide range of libraries that also make use of Apache Arrow, like Pandas and DuckDB.
```

Comment:
Moving the link to the next rather than the title is probably a good idea - let's do the same for plotly / duckdb?

Reply:
I think it is nicer if we move the link out of the title. I'll update